nft_ingester/src/program_transformers/bubblegum/mint_v1.rs

```diff
@@ -86,92 +89,38 @@ where
-                    chain_data_mutability: Set(chain_mutability),
-                    chain_data: Set(chain_data_json),
-                    metadata_url: Set(uri.clone()),
-                    metadata: Set(JsonValue::String("processing".to_string())),
-                    metadata_mutability: Set(Mutability::Mutable),
-                    slot_updated: Set(slot_i),
-                    reindex: Set(Some(true)),
-                    raw_name: Set(name.to_vec()),
-                    raw_symbol: Set(symbol.to_vec()),
-                    ..Default::default()
-                };
-
-                let mut query = asset_data::Entity::insert(data)
-                    .on_conflict(
-                        OnConflict::columns([asset_data::Column::Id])
-                            .update_columns([
-                                asset_data::Column::ChainDataMutability,
-                                asset_data::Column::ChainData,
-                                asset_data::Column::MetadataUrl,
-                                asset_data::Column::MetadataMutability,
-                                asset_data::Column::SlotUpdated,
-                                asset_data::Column::Reindex,
-                            ])
-                            .to_owned(),
-                    )
-                    .build(DbBackend::Postgres);
-                query.sql = format!(
-                    "{} WHERE excluded.slot_updated > asset_data.slot_updated",
-                    query.sql
-                );
-                txn.execute(query)
-                    .await
-                    .map_err(|db_err| IngesterError::AssetIndexError(db_err.to_string()))?;
-                // Insert into `asset` table.
-                let delegate = if owner == delegate || delegate.to_bytes() == [0; 32] {
-                    None
-                } else {
-                    Some(delegate.to_bytes().to_vec())
-                };
-                let tree_id = bundle.keys.get(3).unwrap().0.to_vec();
-
-                // Set initial mint info.
-                let asset_model = asset::ActiveModel {
-                    id: Set(id_bytes.to_vec()),
-                    owner_type: Set(OwnerType::Single),
-                    frozen: Set(false),
-                    tree_id: Set(Some(tree_id.clone())),
-                    specification_version: Set(Some(SpecificationVersions::V1)),
-                    specification_asset_class: Set(Some(SpecificationAssetClass::Nft)),
-                    nonce: Set(Some(nonce as i64)),
-                    royalty_target_type: Set(RoyaltyTargetType::Creators),
-                    royalty_target: Set(None),
-                    royalty_amount: Set(metadata.seller_fee_basis_points as i32), //basis points
-                    asset_data: Set(Some(id_bytes.to_vec())),
-                    slot_updated: Set(Some(slot_i)),
-                    ..Default::default()
-                };
+                unprotected_upsert_asset_data(
```

Comment:
Is unprotected only possible here because of `!asset_should_be_updated` from earlier?

Reply:
Fixed in https://github.com/metaplex-foundation/digital-asset-rpc-infrastructure/pull/134